src/server/fetch/fetch.js

```diff
@@ -0,0 +1,44 @@
+import { Agent } from 'https';
+import fetch from 'node-fetch';
+import getCerts from './certs';
+
+let agentMemo = null;
+
+const getAgent = async () => {
+  const certsMemo = await getCerts();
+  const { certChain, key, ca, caPath, keyPath, certChainPath } = certsMemo;
+
+  if (!ca) throw new Error(`No CA Bundle found at ${caPath}`);
+  if (!certChain)
+    throw new Error(`No Public Key Chain found at ${certChainPath}`);
+  if (!key) throw new Error(`No Private Key found at ${keyPath}`);
+
+  const agentOptions = { cert: certChain, key, ca };
+  agentMemo = new Agent(agentOptions);
+  return agentMemo;
+};
+
+const fetchWithCerts = async requestUrl => {
+  const agent = await getAgent();
+
+  const opts = {
+    method: 'GET',
+    agent,
+    headers: {
+      Accept: 'application/json',
+    },
+  };
+
+  try {
+    return await fetch(requestUrl, opts);
+  } catch (error) {
+    const socketConnectionFailedErrors = ['ECONNRESET', 'EPIPE', 'ETIMEDOUT'];
+    if (socketConnectionFailedErrors.includes(error.code)) {
+      return fetch(requestUrl, opts);
```

Comment:
I know this code came from somewhere else but this retry is a bit hacky, I think we should start without it and see if we need it. We need to log the errors so we can decide if we need this sort of logic ourselves

Reply:
Agreed. Forgot to rip it out last time.